pkg/preflight/utils.go

```diff
@@ -9,6 +9,16 @@ func IsEKS() bool {
 	return Instance().ProviderName() == string(cloudops.AWS) && Instance().K8sDistributionName() == eksDistribution
 }
 
+// IsGKE returns whether the cloud environment is running GKE
+func IsGKE() bool {
+	return Instance().ProviderName() == string(cloudops.GCE) && Instance().K8sDistributionName() == gkeDistribution
+}
+
+// IsAzure() returns whether the cloud environment is running Azure
+func IsAzure() bool {
```

Comment:
I believe we don't need this after the above comment. The ProviderName already returns the azure provider. In future we can have a method to check if AKS or not.

Reply:
Yes, that makes sense.